utils/githooks/find_base.sh

```diff
@@ -23,9 +23,22 @@ if command -v gh > /dev/null 2>&1; then
         fi
     fi
 else
-    # With no 'gh' command installed then check against origin/master.
+    # With no 'gh' command installed, use the "closest" branch as the target,
+    # calculated as the sum of the commits this branch is ahead and behind.
     # shellcheck disable=SC2034
-    TARGET="$ORIGIN"/master
+    all_bases=("master" "release/2.4" "feature/cat_recovery" "feature/multiprovider")
```

Comment:
You might want to run `git branch -rl release/*` to get a more complete list here.

Reply:
What is that supposed to do? It gives me empty output

Reply:
Try `git branch --list -r "origin/release/*"`

I'm thinking it'll give you a list of remote branches using a pattern.

Reply:
Actually, I give up. I can't get this to work AND appease the lint check at the same time, because bash is so antiquated it's nearly impossible to get the syntax right